pkg/util/util.go

```diff
@@ -179,3 +181,40 @@ func CheckWithRegex(pattern, name string) bool {
 
 	return reg.MatchString(name)
 }
+
+func GetMappedGitOpsComponent(component appstudiov1alpha1.Component) gitopsgenv1alpha1.Component {
+	gitopsMapComponent := &gitopsgenv1alpha1.Component{}
+
+	gitopsMapComponent = &gitopsgenv1alpha1.Component{
```

Comment:
```suggestion
	gitopsMapComponent = gitopsgenv1alpha1.Component{
```

and you can just return like `return gitopsMapComponent`

Reply:
Done.